apps/linear_algebra/src/halide_blas.cpp

```diff
@@ -0,0 +1,331 @@
+#include <string.h>
+#include "halide_blas.h"
+
+#define assert(cond) halide_assert(0,cond)
```

Comment:
This is a problem if someone wants to use this in a codebase that actually makes use of user_context. (Really a separate, known bug, but still...)